### ISSUE TITLE
Unused fields on the conversation schema

### ISSUE THREAD

**Issue by benedict-tshaba** [tattersoftware/codeigniter4-chat#24]
Hi, this is more of a question than a anything. I see we have the 'private' and 'direct' fields on the conversations table, but these are not being used anywhere. (as far as I could search).

We are trying to implement a sort of 'private chat' functionality on our system, and are currently looking at using this library/module. Just wondering if these fields have a purpose (might be something to be implemented in future). Or if we can re-purpose them for our use-case?

The idea being any changes we make, we would like to push them back into this project for everyone's benefit.

**Comment by MGatner** [comment]
I don't believe they are in use, and if you do want to submit a feature enhancement I would happily consider it for release!
